common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java

```diff
@@ -0,0 +1,327 @@
+        RuntimeMXBean runtimeMXBean = PowerMockito.mock(RuntimeMXBean.class);
+        PowerMockito.when(runtimeMXBean.getName()).thenReturn("40@abc.com");
+        PowerMockito.doReturn(runtimeMXBean).when(ManagementFactory.class);
+        ManagementFactory.getRuntimeMXBean();
+
+        Assert.assertEquals(40, UtilAll.getPid());
+    }
+
+    @PrepareForTest({RuntimeMXBean.class,
+            UtilAll.class, ManagementFactory.class})
+    @Test
+    public void testGetPid2() {
+        PowerMockito.mockStatic(ManagementFactory.class);
+        RuntimeMXBean runtimeMXBean = PowerMockito.mock(RuntimeMXBean.class);
+        PowerMockito.when(runtimeMXBean.getName()).thenReturn(null);
+        PowerMockito.doReturn(runtimeMXBean).when(ManagementFactory.class);
+        ManagementFactory.getRuntimeMXBean();
+
+        Assert.assertEquals(-1, UtilAll.getPid());
+    }
+
+    @PrepareForTest({UtilAll.class})
+    @Test
+    public void testCurrentStackTrace() {
+        PowerMockito.mockStatic(Thread.class);
+        StackTraceElement stackTraceElement =
+                new StackTraceElement("Bar", "foo", "baz", 101);
+        PowerMockito.when(Thread.currentThread().getStackTrace())
+                .thenReturn(new StackTraceElement[]{stackTraceElement});
+
+        Assert.assertEquals("\n\tBar.foo(baz:101)",
+                UtilAll.currentStackTrace());
+    }
+
+    @Test
+    public void testOffset2FileName() {
+        Assert.assertEquals("00000000000000000002",
+                UtilAll.offset2FileName(2l));
+        Assert.assertEquals("00000001558787014000",
+                UtilAll.offset2FileName(1558787014000l));
+    }
+
+    @PrepareForTest({UtilAll.class, System.class})
+    @Test
+    public void testComputeEclipseTimeMilliseconds() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1558787014000l);
+
+        Assert.assertEquals(2592000000l,
+                UtilAll.computeEclipseTimeMilliseconds(1556195014000l));
+    }
+
+    @PrepareForTest({Calendar.class, UtilAll.class})
+    @Test
+    public void testIsItTimeToDo() {
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTimeInMillis(1558787014000l);
+        PowerMockito.mockStatic(Calendar.class);
+        PowerMockito.when(Calendar.getInstance()).thenReturn(calendar);
+
+        Assert.assertTrue(UtilAll.isItTimeToDo("13;12;14"));
+
+        Assert.assertFalse(UtilAll.isItTimeToDo("00;00;00;"));
+    }
+
+    @PrepareForTest({UtilAll.class, System.class})
+    @Test
+    public void testTimeMillisToHumanString() {
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1558787014000l);
+
+        Assert.assertEquals("20190525132334000",
+                UtilAll.timeMillisToHumanString());
+    }
+
+    @Test
+    public void testTimeMillisToHumanStringLong2() {
```

Comment:
Why the "2" on the end of the test method name? I didn't see a testTimeMillisToHumanStringLong()

Reply:
Done